ui/imports/shared/controls/chat/ChatInputLinksPreviewArea.qml

```diff
@@ -107,6 +117,7 @@ Control {
                         onClose: root.linkPreviewModel.removePreviewData(d.filteredModel.mapToSource(index))
```

Comment:
I know it's not part of that pr, but assuming some methods on models is quite unusual and bit problematic. Cleaner solution is to emit signal here (like e.g. `previewDataRemoveRequested`, called with mapped index) instead of calling `removePreviewData` and handling removal externally.

Even now, in storybook we have:
```
file:///home/michalc/workspace/status-desktop/ui/imports/shared/controls/chat/ChatInputLinksPreviewArea.qml:117: TypeError: Property 'removePreviewData' of object QQmlListModel(0x55cd0fb11270) is not a function
```
when clicking "X" on an item